CHANGELOG.md

```diff
@@ -1,3 +1,54 @@
+# v1.29.0 Release - 07/30/2021
+Note: This release comes with a new config version, `v2beta20`. To upgrade your skaffold.yaml, use `skaffold fix`. If you choose not to upgrade, skaffold will auto-upgrade as best as it can.
+
+Highlights:
+* add documentation for `lifecycle hooks` [#6330](https://github.com/GoogleContainerTools/skaffold/pull/6330)
```

Comment:
How about: `Introducing Skaffold lifecycle hooks`?

Reply:
Done- - now "Introducing Skaffold lifecycle hooks!"

Reply:
I don't think this was done